dcrtimed/backend/filesystem/filesystem.go

```diff
@@ -830,8 +827,6 @@ func New(root, cert, host string, enableCollections bool, passphrase []byte) (*F
 	}
 	fs.enableCollections = enableCollections
 
-	// Runtime bits
-	dcrtimewallet.UseLogger(log)
```

Comment:
I misunderstood what you were trying to do before.  My mistake.  This should be left in.  It's a package in dcrtime so it needs logging.